### ISSUE TITLE
Migrate the existing View Pager to Pager 

### ISSUE THREAD

**Issue by angrezichatterbox** [scribe-org/Scribe-Android#256]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/scribe-org/Scribe-Android/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow Scribe-Android's [Code of Conduct](https://github.com/scribe-org/Scribe-Android/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

The app currently employs a View Pager, and the compose views are being set from fragments. This approach was implemented to ensure a functional main throughout the migration process of individual pages to Jetpack Compose. Once the migration is complete, we can convert the View Pager to a Compose alternative that uses the Pager layout. This issue addresses this conversion.

**Note:** This issue is blocked by #248, #217, which will migrate the remaining XML layouts to Jetpack Compose.

For more details on resolving this issue, refer to the documentation for [Pager](https://developer.android.com/develop/ui/compose/layouts/pager). 

### Contribution

I would love to work on this and am more than happy to help anyone solve this issue :)

**Comment by Saifuddin53** [comment]
@andrewtavis @angrezichatterbox can I work on this after my exams are over tomorrow?

**Comment by angrezichatterbox** [comment]
Yes it is totally fine. I would assign the issue to you. Thanks :)

**Comment by angrezichatterbox** [comment]
Closed by #281. Thanks for your work @Saifuddin53
